src/libraries/Microsoft.PowerFx.Core/IR/Symbols/GlobalSymbol.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.PowerFx.Core.Types;
+using Microsoft.PowerFx.Types;
+
+namespace Microsoft.PowerFx.Core.IR.Symbols
```

Comment:
Should these be in the `IR` folder/bamespace? It seems like they're closer to Binding or Entities to me.

Reply:
I agree, this is more logical in Binding ns